### ISSUE TITLE
Store Worker Details in Redis for metrics / observability 

### ISSUE THREAD

**Issue by lukepolo** [taskforcesh/bullmq-pro-support#67]
It would be awesome to get the worker details , things like the WorkerOptions.  We have a lot of workers and they may spin up and down depending on load. Also we change our concurrency based on CPU load / Memory Load etc. 

Would be awesome if we could change the concurrency from redis as well !

**Comment by manast** [comment]
Yes, we have some ideas to improve the data that we can get from the workers, for instance which worker is processing which job, and other stats. Not sure when we will be able to fit in such a feature though, but it is definitively going to happen.

**Comment by roggervalf** [comment]
concurrency can be saved but at global queue level with https://docs.bullmq.io/guide/queues/global-concurrency
